readme.txt

```diff
@@ -91,6 +91,12 @@ Add this to your theme's functions.php.
 
 	$tsml_street_only = false;
 
+= Can I add a feedback_url to each meeting when using tsml_ui? = Add a URL to your themes functions.php.
```

Comment:
it would be good to refer to it as `TSML UI` for consistency

Reply:
Good point. Change made.